src/stores/firebase.js

```diff
@@ -73,20 +73,24 @@ export const useFirebaseStore = defineStore("firebase", {
         this.firebaseApp = initializeApp(firebaseConfigProduction);
       }
 
+      this.firestore = initializeFirestore(this.firebaseApp,{});
+      this.auth = initializeAuth(this.firebaseApp,{});
+      this.functions = getFunctions(this.firebaseApp, "europe-west1");
```

Comment:
In general stuff seems to be working, however when trying the functions emulation I currently encounter 2 erros.
1. I guess I need to set up an API key for algolia? Which I guess is saved in gcloud?
```
Error when performing Algolia index TypeError [ERR_HTTP_INVALID_HEADER_VALUE]: Invalid value \"undefined\" for header \"x-algolia-api-key\"
at ClientRequest.setHeader (node:_http_outgoing:579:3)
at new ClientRequest (node:_http_client:256:14)
at new request (node:https:353:10)
at Object.obj.<computed> [as request] (/home/vmasarik/git/deleteme/activisthandbook-editor/node_modules/firebase-tools/lib/emulator/functionsEmulatorRuntime.js:233:31)
at /home/vmasarik/.cache/firebase/extensions/algolia/firestore-algolia-search@0.5.14/functions/node_modules/@algolia/requester-node-http/dist/requester-node-http.cjs.js:33:72
at new Promise (<anonymous>)
at Object.send (/home/vmasarik/.cache/firebase/extensions/algolia/firestore-algolia-search@0.5.14/functions/node_modules/@algolia/requester-node-http/dist/requester-node-http.cjs.js:18:20)
at retry (/home/vmasarik/.cache/firebase/extensions/algolia/firestore-algolia-search@0.5.14/functions/node_modules/@algolia/transporter/dist/transporter.cjs.js:222:38)
at /home/vmasarik/.cache/firebase/extensions/algolia/firestore-algolia-search@0.5.14/functions/node_modules/@algolia/transporter/dist/transporter.cjs.js:235:16
at processTicksAndRejections (node:internal/process/task_queues:96:5) {
code: 'ERR_HTTP_INVALID_HEADER_VALUE'
```

2. This is way larger error, eg 300 lines, but I guess I need a proper login for cloudfare as well? Eg. the credentials are in Gcloud, and not locally set. See the `undefined` authorization token.
```
"Unhandled error [AxiosError: Request failed with status code 404] {
    code: 'ERR_BAD_REQUEST',
  config: {
      transitional: {
        silentJSONParsing: true,
      forcedJSONParsing: true,
      clarifyTimeoutError: false
    },
    adapter: [Function: httpAdapter],
    transformRequest: [ [Function: transformRequest] ],
    transformResponse: [ [Function: transformResponse] ],
    timeout: 0,
    xsrfCookieName: 'XSRF-TOKEN',
    xsrfHeaderName: 'X-XSRF-TOKEN',
    maxContentLength: -1,
    maxBodyLength: -1,
    env: { FormData: [Function] },
    validateStatus: [Function: validateStatus],
    headers: {
        Accept: 'application/json, text/plain, */*',
      'Content-Type': 'multipart/form-data; boundary=--------------------------392168075499339166600082',
      Authorization: 'Bearer undefined',
      'User-Agent': 'axios/0.27.2',
      'content-type': 'multipart/form-data; boundary=--------------------------392168075499339166600082'
    },
    method: 'post',
    url: 'https://api.cloudflare.com/client/v4/accounts/undefined/images/v2/direct_upload',
    data: FormData {
        _overheadLength: 107,
      _valueLength: 15,
```